ironic-scripts/runironic-conductor

```diff
@@ -13,4 +13,12 @@ until ironic-dbsync --config-file /etc/ironic/ironic.conf upgrade; do
   echo "WARNING: ironic-dbsync failed, retrying"
   sleep 1
 done
+
+if [ "$IRONIC_TLS_SETUP" == "true" ]; then
+    inotifywait -m -e delete_self "${IRONIC_CERT_FILE}" | while read file event; do
+     kill $(pgrep ironic)
+     /usr/bin/ironic-conductor &
```

Comment:
No starting anything in the background please. The previous line will likely cause the container to stop anyway.